sr_object_symmetry/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sr_object_symmetry</name>
+  <version>0.0.0</version>
+  <description>The sr_object_symmetry package</description>
+
+  <maintainer email="software@shadowrobot.com">Shadow Robot's software team</maintainer>
+
+  <license>Private</license>
+
+  <author>Shadow Robot's software team</author>
```

Comment:
Maybe we could also add the original author in a second author tag.

Reply:
done